PrebidMobile/Rendering/PrebidMobileRendering/PrebidMobileRendering/Prebid/PBMCacheRenderers/MediaView.swift

```diff
@@ -0,0 +1,288 @@
+//
+//  PBMMediaViewT.swift
+//  PrebidMobileRendering
+//
+//  Copyright © 2021 Prebid. All rights reserved.
+//
+
+import UIKit
+
+ let DEFAULT_VIEWABILITY_POLLING_INTERVAL: TimeInterval = 0.2
+ let IGNORE_CLICKS_IF_UNREGISTERED = true
+
+ enum MediaViewState {
+    case undefined, playbackNotStarted, playing, pausedByUser, pausedAuto, playbackFinished
+}
+
+@objc(PBMMediaView) public class MediaView: UIView, PBMPlayable, PBMAdViewManagerDelegate {
```

Comment:
Not sure here
- Looks like we can avoid `@objc` for availability to  ObjC since it is inherited from UIView. 
- PBM is not a desirable prefix. Prebid is better if needed. 

Which problems we can face with if we totally remove @objc(PBMMediaView) ?